src/axom/inlet/SphinxDocWriter.cpp

```diff
@@ -36,50 +36,35 @@ void SphinxDocWriter::writeDocumentation() {
   m_oss << ".. |uncheck|    unicode:: U+2610 .. UNCHECKED BOX" << std::endl;
   m_oss << ".. |check|      unicode:: U+2611 .. CHECKED BOX" << std::endl;
   writeDocumentationHelper(m_sidreRootGroup);
-  m_rstTables.push_back(m_currentTable);
+  // m_rstTables.push_back(m_currentTable);
   writeAllTables();
   m_outFile.open(m_fileName);
   m_outFile << m_oss.str();
   m_outFile.close();
 }
 
 void SphinxDocWriter::writeDocumentationHelper(axom::sidre::Group* sidreGroup) {
-  SLIC_ASSERT_MSG(sidreGroup, "Root was nullptr");
+  SLIC_WARNING_IF(!sidreGroup, "Root was nullptr");

```

Comment:
Clarify this message: Inlet's Sidre root group was not defined while writing documentation